### DISCUSSION TITLE
How to check for optional argument?

### DISCUSSION THREAD

**Discussion by kelvinkoko** [ohmjs/ohm#382] [Help] [answered]
Sorry it maybe a simple question, but i seems cannot find the answer in doc
For example i have this grammar
`If = "if" StatementInRound StatementInCurly Else?`
The `Else` is optional,
so in semantics, what is the recommend way to check whether this match contain "Else " case?
``` 
// Semantic section: 
If(_if, condition, yesStatements, elseStatements) {
// How i can check for whether elseStatements is  included or not?  
}
```

**Comment by gmhta** [comment, marked as answer]
```
elseStatements.numChildren === 0
```
